azure-monitor-opentelemetry/README.md

```diff
@@ -53,11 +53,19 @@ pip install azure-monitor-opentelemetry --pre
+* resource - Specified the OpenTelemetry [resource][opentelemetry_spec_resource] associated with your application. See [this][ot_sdk_python_resource] for default behavior. This include configuration via environment variables.
+* disable_logging - If set to `True`, disables collection and export of logging telemetry. Can also be set via the `APPLICATIONINSIGHTS_DISABLE_LOGGING` environment variable. Defaults to `False`.
+* disable_metrics - If set to `True`, disables collection and export of metric telemetry. Can also be set via the `APPLICATIONINSIGHTS_DISABLE_METRICS` environment variable. Defaults to `False`.
+* disable_tracing - If set to `True`, disables collection and export of distributed tracing telemetry. Can also be set via the `APPLICATIONINSIGHTS_DISABLE_TRACING` environment variable. Defaults to `False`.
+* logging_level - Specifies the [logging level][logging_level] of the logs you would like to collect for your logging pipeline. Can also be set via the `OTEL_LOG_LEVEL` environment variable. This environment variable should be set to the number that represents the log leval, not a string representing that log level's name. Defaults to 0 which is logging.NOTSET.
+* logger_name = Specifies the [logger name][logger_name_hierarchy_doc] under which logging will be instrumented. Can also be set via the `APPLICATIONINSIGHTS_LOGGER_NAME` environment variable. Defaults to "" which corresponds to the root logger.
+* logging_export_interval_ms - Specifies the logging export interval in milliseconds. Can also be set via the `OTEL_BLRP_SCHEDULE_DELAY` environment variable. Defaults to 5000.
+* metric_readers - Specifies the [metric readers][ot_metric_reader] that you would like to use for your metric pipeline. Accepts a list of [metric readers][ot_sdk_python_metric_reader].
+* views - Specifies the list of [views][opentelemetry_spec_view] to configure for the metric pipeline. See [here][ot_sdk_python_view_examples] for example usage.
+* sampling_ratio - Specifies the ratio of distributed tracing telemetry to be [sampled][application_insights_sampling]. Accepted values are in the range [0,1]. Can also be set with the `OTEL_TRACES_SAMPLER_ARG` environment variable. Defaults to 1.0, meaning no telemetry is sampled out.
+* tracing_export_interval_ms - Specifies the distributed tracing export interval in milliseconds. Can also be set via the `APPLICATIONINSIGHTS_TRACING_EXPORT_INTERVAL_MS` environment variable. Defaults to 5000.
+* instrumentation_config - Specifies a dictionary of kwargs that will be applied to instrumentation configuration. You can specify which instrumentation you want to configure by name in the key field and value as a dictionary representing `kwargs` for the corresponding instrumentation. Can also be set via the `APPLICATIONINSIGHTS_INSTRUMENTATION_CONFIG` environment variable.
```

Comment:
This might not work since instrumentation configuration values could be values that aren't strings, and env vars only contain string values. We would need ways to smartly parse string values for this to work. Might be a bit too complicated to support right now.